packages/state-transition/src/epoch/processPendingBalanceDeposits.ts

```diff
@@ -14,8 +13,8 @@ import {getCurrentEpoch} from "../util/epoch.js";
  * TODO Electra: Update ssz library to support batch push to `pendingBalanceDeposits`
  */
 export function processPendingBalanceDeposits(state: CachedBeaconStateElectra, cache: EpochTransitionCache): void {
+  const nextEpoch = state.epochCtx.epoch + 1;
```

Comment:
I really like the name `upcomingEpoch` made by @matthewkeil  for `processEpoch` code in his async shuffling PR #6938

Reply:
yea name is good but i think good to be closer to naming used in specs, though not a strong opinion

Reply:
+1 on following spec naming, I think we should have a good reason if we wanna deviate from terms used in spec.

Reply:
Yeah, if we really have a problem with the naming, we should aim to update it in the spec instead. Deviating off standards create more context loss for new people reading our code IMO.